src/DynamoPackagesUI/Utilities/CefCommands.cs

```diff
@@ -36,11 +39,12 @@ public string SessionData
 
         public PackageManagerViewModel PackageMgrViewMdodel { get; set; }
 
-        public CefHelper(DynamoViewModel dynamoViewModel, PackageLoader model, PackageManagerViewModel packageMgrViewModel)
+        public CefCommands(DynamoViewModel dynamoViewModel, PackageLoader model, PackageManagerViewModel packageMgrViewModel)
```

Comment:
The variable name could be `viewModel` and also the corresponding property could be `ViewModel` or `PackageManagerViewModel`